routes/authentication_cr.js

```diff
@@ -0,0 +1,81 @@
+const express = require("express");
+const router = express.Router()
+const bcrypt = require("bcryptjs")
+
+//importing local files
+const db = require("../db/models");
+const { environment } = require("../config");
+const { asyncHandler, csrfProtection, signUpValidator, validationResult, loginValidators } = require("./utiles");
+
+//Sing-Up
+
+router.post("/sign-up", csrfProtection, signUpValidator, asyncHandler(async(req, res) => {
+
+    const { username, email, password } = req.body
+
+    const validatorError = validationResult(req)
+    if(validatorError.isEmpty()) {
+        const hashedPassword = await bcrypt.hash(password, 10);
+        await db.User.create({
+            username,
+            email,
+            password: hashedPassword,
+            createdAt: new Date(),
+            updatedAt: new Date()
+        });
+
+
+        res.json({}); // Authentication Token to be inserted in the future
+
+
+    } else {
+        const errors = validatorError.array().map((error) => error.msg);
+        res.json({ errors });
+    }
+}))
+
+router.get("/demo", asyncHandler( async (req, res) => {
+    const user = await db.User.findOne({
+        where: {
+            username: "DemoUser",
+            email: "DemoUser@NatlRoutes.com"
+        }
+    });
+    console.log(user);
```

Comment:
Try to remove `console.log`s before merging to your main branch. Don't let noisy logging build up or be seen by production users!